### DISCUSSION TITLE
Disable email notifications

### DISCUSSION THREAD

**Discussion by Modrx** [mastodon/mastodon#28897] [Q&A]
I would like to disable the following as default for new signups:

Under Settings > Preferences > Notifications

Events for e-mail notifications
```
-Someone followed you
-Someone requested to follow you
-Someone mentioned you
```
Can you please tell me what file to edit to disable all email notifications.

Thank you!

**Comment by robertpaessler** [comment]
I would like to join here:
Can I turn off getting an email as an admin when there is a new account to review?

**Comment by ikanor** [comment]
We faced the exact same issue. We have just setup a server and we have some users with +1000 followers migrating, which is causing or e-mail quota to dry up when they migrate while not being logged in on the app. We have been looking for a way to change de default email settings, or to deactivate the email sending while running a migration, but we have not found anything. 

We could of course change change the hardcoded default settings [here](https://github.com/mastodon/mastodon/blob/2f98134ac69ee840095c9d8389e4b2fff72f20c1/app/models/user_settings.rb#L39), but it doesn’t seem like a very good idea to fork one of the core classes.
